app/controllers/assessments_controller.rb

```diff
@@ -991,11 +991,19 @@ def valid_asmt_tar(tar_extract)
       pathname.chomp!("/") if entry.directory?
       # nested directories are okay
       if entry.directory? && pathname.count("/") == 0
-        return false if asmt_name
+        if asmt_name
+          flash[:error] = "Error in tarball: Found root directory #{asmt_name}
+                           but also encountered root directory #{pathname} folder. Ensure
```

Comment:
Nit: redundant "folder" after `... also encountered root directory #{pathname}`

Reply:
For consistency, would prefer
"... Found root directory ... but also [found] root directory ..."
"... only one root directory in the [tarball]"